src/octoprint/timelapse.py

```diff
@@ -498,7 +500,7 @@ def __init__(self, post_roll=0, fps=25):
 
         self._capture_dir = settings().getBaseFolder("timelapse_tmp")
         self._movie_dir = settings().getBaseFolder("timelapse")
-        self._snapshot_url = settings().get(["webcam", "snapshot"])
+        self._snapshot_url = get_default_webcam().snapshot
```

Comment:
Can be, but then the preconditions in line 411 are not met and it's not started. Added a clean Exception anyways just to be sure in case the defaultWebcam is removed during a timelapse recording